components/brave_wallet_ui/page/screens/composer_ui/token_list_item/token_list_item.tsx

```diff
@@ -67,151 +74,212 @@ export const TokenListItem = React.forwardRef<HTMLDivElement, Props>(
-                />
-              )}
-              {tokensNetwork &&
-                checkIfTokenNeedsNetworkIcon(
-                  tokensNetwork,
-                  token.contractAddress
-                ) && (
-                  <NetworkIconWrapper>
-                    <CreateNetworkIcon
-                      network={tokensNetwork}
-                      marginRight={0}
-                    />
-                  </NetworkIconWrapper>
-                )}
-            </IconsWrapper>
-            <Column alignItems='flex-start'>
-              <Row width='unset'>
-                <NameAndBalanceText
-                  textSize='14px'
-                  isBold={true}
-                  textAlign='left'
-                >
-                  {tokenDisplayName}
-                </NameAndBalanceText>
-                {disabledText && (
-                  <>
-                    <HorizontalSpace space='8px' />
-                    <DisabledLabel>{getLocale(disabledText)}</DisabledLabel>
-                  </>
-                )}
-              </Row>
-              <NetworkAndFiatText
+        {label &&
+          (label === 'owned' ? (
+            <Row
+              justifyContent='space-between'
+              padding='10px 16px'
+            >
+              <Text
                 textSize='12px'
                 isBold={false}
-                textAlign='left'
+                textColor='secondary'
               >
-                {networkDescription}
-              </NetworkAndFiatText>
-            </Column>
-          </IconAndName>
-          {balance && !token.isErc721 && !token.isNft && (
-            <Column alignItems='flex-end'>
-              <NameAndBalanceText
-                textSize='14px'
-                isBold={true}
-                textAlign='right'
+                {getLocale('braveWalletOwned')}
+              </Text>
+              <Text
+                textSize='12px'
+                isBold={false}
+                textColor='secondary'
               >
-                {formatTokenBalanceWithSymbol(
-                  balance,
-                  token.decimals,
-                  token.symbol
+                {getLocale('braveWalletAmount24H')}
+              </Text>
+            </Row>
+          ) : (
+            <Row
+              justifyContent='flex-start'
+              padding='10px 16px'
+            >
+              <Text
+                textSize='12px'
+                isBold={false}
+                textColor='secondary'
+              >
+                {getLocale('braveWalletNotOwned')}
+              </Text>
+            </Row>
+          ))}
+        <ButtonWrapper
+          width='100%'
+          justifyContent='space-between'
+        >
+          <Button
+            onClick={onClick}
+            disabled={!!disabledText}
+          >
+            <IconAndName justifyContent='flex-start'>
+              <IconsWrapper>
+                {token.asset.isErc721 || token.asset.isNft ? (
```

Comment:
perf-nit: might run faster if we check `isNft` first

Reply:
Moved!